js/app.js

```diff
@@ -68,7 +68,7 @@ function CreateStore(name,minCust,maxCust,avgCookies) {
 
 function createTable() {
   var table = document.createElement('table');
-  body.appendChild(table);
+  section.appendChild(table);
   table.id = 'bodyTable';
```

Comment:
This `id` is almost destined to become confusing. Why not just call it `sales-table`? Especially because a few lines below this you have another `id` called `tableBody`